pkgs/development/libraries/physics/hepmc3/default.nix

```diff
@@ -16,9 +24,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-zQ91yA91VJxZzCqCns52Acd96Xyypat1eQysjh1YUDI=";
   };
 
-  nativeBuildInputs = [ cmake ];
-  buildInputs = [ root_py ]
-    ++ lib.optional withPython python;
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    nlohmann_json # Needed since root 6.26.10
```

Comment:
Move to `root.propagatedBuildInputs`?

Reply:
Specifying it in `root.propagatedBuildInputs` alone doesn't eliminate the CMake error when building hepmc3.

Should we inherit the `propagatedBuildInputs` from root in hepmc/default.nix?